### PR TITLE
HW1 for review

### PR DESCRIPTION
**Pull request by dubinchuk**
First JS homework for approval

### CODE REVIEW

File: JS_HW_1/JS_HW_1.3.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// Task3**. Напишите программу, которая принимает целое положительное число n (одно любое число от 1 до 9), 
+// и выводит сумму равную n + nn + nnn, где n не перемножаются, а конкатенируются
+
+let n = '';
```
Comment (VladimirLQA) [inline JS_HW_1/JS_HW_1.3.js:4]: What is the purpose of this variable ?

File: JS_HW_1/JS_HW_1.3.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+const nSum = function(n){
+    if (n >= 1 && n <= 9 && n % 1 === 0) {
+        let nString = String(n);
+        let result = Number(nString) + Number(nString + nString) + Number(nString + nString + nString);
```
Comment (VladimirLQA) [inline JS_HW_1/JS_HW_1.3.js:8]: Use `const` for values that won't change further

**Review by VladimirLQA — CHANGES_REQUESTED** [review]

**Comment by VladimirLQA** [comment]
Provide description for PR, please

**Review by VladimirLQA — APPROVED** [review]

File: JS_HW_1/JS_HW_1.3.js (hunk 1 above)
Comment (dubinchuk) [reply JS_HW_1/JS_HW_1.3.js]: Fixed the mistake: `n` is a function parameter, no need to declare it as a variable

File: JS_HW_1/JS_HW_1.3.js (hunk 2 above)
Comment (dubinchuk) [reply JS_HW_1/JS_HW_1.3.js]: Got it. Fixed

**Review by anatoly-karpovich — APPROVED** [review]

**Pull request merged**